worlds/bk_sudoku/docs/fr_Sudoku.md

```diff
@@ -0,0 +1,13 @@
+# Bk Sudoku
+
+## Qu'est-ce que ce jeu ?
+
+BK Sudoku n'est pas un jeu typique d'Archipelago ; il s'agit plutôt d'un client générique de Sudoku qui peut se connecter à n'importe quel multi-monde existant. Une fois connecté, vous pouvez jouer au Sudoku pour débloquer des indices aléatoires pour votre jeu. Bien que lent, il vous donnera quelque chose à faire lorsque vous n'arrivez pas à atteindre les vérifications dans votre jeu.
```

Comment:
I believe that the concept of random hint does not translate well. I suggest to drop the word "aléatoires" here and leave the heavy lifting to the next section.

"Bien que lent" could lead to confusion since it is not that it make progress slow. It is more that you can't expect that the time investment is worth it? I do not have a suggestion at the moment... :(

I believe that translation of technical terms like "check" can be confusing. Suggestion: change "n'arrivez pas à atteindre les vérifications dans votre jeu." for "ne pouvez plus faire de progrès dans votre jeu."